app/controllers/admin/users_controller.rb

```diff
@@ -1,8 +1,10 @@
-class Admin::UsersController < Admin::ApplicationController
-  def index
-    ransack_params = params[:q]
-    @q = User.search(ransack_params)
-    @users = @q.result(distinct: true).
-             paginate(page: params[:page], per_page: 20)
+module Admin
+  class UsersController < Admin::BaseController
+    def index
+      ransack_params = params[:q]
+      @q = User.search(ransack_params)
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name '@q' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->